src/Fake-netcore.sln

```diff
@@ -1,73 +1,72 @@
-
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26124.0
```

Comment:
I'm pretty sure something in here makes the build fail on travis... Maybe try to minimize the diff here or revert this change and add changes by hand?

The appveyor failure has something to do with the tests for the legacy FakeLib component. Some behavior was changed?

Reply:
yep not intended to cmmit this file...I will revert